tests/mail/tests.py

```diff
@@ -913,6 +913,26 @@ def test_attach_non_utf8_text_as_bytes(self):
         self.assertEqual(content, b"\xff")
         self.assertEqual(mimetype, "application/octet-stream")
 
+    def test_unicode_attachment_encoding(self):
```

Comment:
```suggestion
    def test_attach_8bit_rfc822_message(self):
```

(make similar to nearby test names)